rpc/pool.go

```diff
@@ -121,12 +122,19 @@ func (p *SessionPool) getSessionFromPool(id proto.NodeID) (sess *Session, ok boo
 	return
 }
 
+func (p *SessionPool) Get(id proto.NodeID) (net.Conn, error) {
```

Comment:
**[golint]** <sub>reported by [reviewdog](https://github.com/haya14busa/reviewdog) :dog:</sub>
exported method SessionPool.Get should have comment or be unexported